html.md

```diff
@@ -11,10 +11,31 @@ We use HAML at The Frontier Group, ERB is only acceptable for mailers, plain tex
 * Use HAML (`-#`) instead of HTML (`/`) comments (exception for conditional comments like below)
 * Use Rails TagHelpers like `image_tag` instead of `%img`
 
+## HAML Initializer
+
+To get more control over HTML doctypes, the default HAML format needs to be set to `:xhtml`.
+This allows us to use the XHTML 1.0 Transitional doctype for email layouts and HTML5 for everything else.
+
+**Meaning:**
+* Email templates doctype: `!!!`
+* Any other layout: `!!! 5`
+
+### Adding the initializer to your project
+
+Copy & paste the following line into `/config/initializers/haml.rb`.
+
+```ruby
+Haml::Template.options[:format] = :xhtml
```

Comment:
Well, afaik `ActiveSupport.on_load(:action_vew)` is not necessary.

In which scenario you will start rails server without loading the module `ActionView`?